tests/test_serialize.py

```diff
@@ -51,17 +51,7 @@
         TensorProdObsC128,
     )
 elif device_name == "lightning.tensor":
-    from pennylane_lightning.lightning_tensor_ops import TensorNetC64, TensorNetC128
-    from pennylane_lightning.lightning_tensor_ops.observables import (
-        HamiltonianC64,
-        HamiltonianC128,
-        HermitianObsC64,
-        HermitianObsC128,
-        NamedObsC64,
-        NamedObsC128,
-        TensorProdObsC64,
-        TensorProdObsC128,
-    )
+    pytest.skip("Lightning Tensor has its own serialize test.", allow_module_level=True)
```

Comment:
Can we change it to "Lightning Tensor seiralization are tested separately in ..."

Reply:
Done!